.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,45 @@
+name: Integration Tests
+on:
+  pull_request:
+    branches:
+      - master
+    types:
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  selenium-tests:
+    name: Selenium Tests
+    runs-on: ubuntu-latest
```

Comment:
I'm not sure if it is worth using a self-hosted runner or not?

Reply:
Maybe a good question for @nebolsinp.

Reply:
I think it's okay to use GH-runners for now

Reply:
actually self-hosted runners might save us about 2-3 minutes in each test run. because of the docker cache.
so the selenium/mysql/nginx docker layers won't be pulled every time from docker ECR, but will be immediately restored from runner's docker cache. same way as we use self-hosted runners on the platform. see the screenshots
<img width="1056" alt="image" src="https://user-images.githubusercontent.com/3922479/234921615-b95b8c6d-966d-44b9-bce5-5c6816a8f9f4.png"> vs 
<img width="1066" alt="image" src="https://user-images.githubusercontent.com/3922479/234923027-9c32b64d-793a-476e-a280-5ccd58f0c271.png">

Reply:
@nebolsinp, would the goal be to use the existing Organization k8s runners? If so, I don't think I have permissions to share those Organization runners with this repo. If not, I don't think I have permission to create new runners in AWS.

Reply:
No need to share runners specifically for that repo. They are shared to the entire organic github org already. just replace `runs-on: ubuntu-latest ` with  `runs-on: self-hosted` and that's it

Reply:
I tried that but the jobs are never picked up. Maybe there is some authorization missing? In any case, I don't see the runners available in the settings [here](https://github.com/orgnc/wordpress-plugin/settings/actions/runners), which seems odd if they are shared.

Reply:
it's because the repository is public. and it's not recommended for self-hosted actions to work in public repositories for security [reasons](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners#self-hosted-runner-security).
<img width="391" alt="image" src="https://user-images.githubusercontent.com/3922479/234990335-860005e7-a894-4ce7-ab0b-75a527e69481.png">

So.. I guess we have to go back to "ubuntu-latest"

Reply:
To be honest I completely forgot that this is a public repo :)

Reply:
Makes sense :) Thanks Pavel!